tools/UpdatePaths/Scripts/74490_ammo_box_to_toolbox_762.txt

```diff
@@ -0,0 +1,3 @@
+#comment This repaths the ammo toolbox (which contains 7.62 stripper clips) to a new path that more adequately explains what it contains
+
+/obj/item/storage/toolbox/ammo : /obj/item/storage/toolbox/a762{@OLD}
```

Comment:
This looks like it should be right, but you still have to run it 😅 
If you did and the map didn't change I guess it isn't correct.

Reply:
ah it's because my branch is outdated hang on >:P